examples/burial-form/ReviewPage.jsx

```diff
@@ -0,0 +1,345 @@
+import React from 'react';
+import {
+  DebuggerView
+} from '@department-of-veterans-affairs/va-forms-system-core';
+import { useFormikContext } from 'formik';
+import { Page } from '@department-of-veterans-affairs/va-forms-system-core';
+import { Link } from 'react-router-dom';
+import { VaOnThisPage } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+const transformFieldValue = (key, field) => {
+  if ( field.value === 'true' | field.value === true ) {
+    return 'Yes';
+  }
+  if (field.value === 'false' | field.value === false ) {
+    return 'No';
+  }
+  if (["from", "to", "veteranDateOfBirth", "deathDate", "burialDate"].indexOf(key) > -1) {
+    const date = new Date(field.value);
+    return date.toLocaleDateString("en-US", { day: 'numeric', month: 'long', year: 'numeric' });
+  }
+  if (["amountIncurred", "amountGovtContribution"].indexOf(key) > -1) {
+    return "$" + field.value
```

Comment:
Getting this ESLint error: `Operands of '+' operation must either be both strings or both numbers. Consider using a template literal. eslint (@typescript-eslint/restrict-plus-operands)`